app/views/email_templates/new.html.slim

```diff
@@ -0,0 +1,34 @@
+h1= t_title(:new)
+
+p Sie können die folgenden Platzhalter benützen:
+ul
+  li %{Anrede}
+  li %{Name}
+  li %{feedback-link}
```

Comment:
`%{EmailLink}`

Reply:
A name that is easy in a hash `{ EmailLink: 'asdfasdf' }` vs `{ 'email-link': 'asdfasdf' }`

Reply:
And I would suggest to have something automatic here, as soon as this functionality is zipped with https://github.com/panter/aoz-003/pull/359

Reply:
Yes, I could temporarily define them in the `EmailTemplate` model.

Reply:
Yes, I can temporarily define them in the `EmailTemplate` model.

Reply:
In the ReminderMailingVolunteer they're right now:
```ruby
[:Anrede, :Einsatz, :Name, :EinsatzStart]
```

Reply:
Why is the `:EinsatzStart` used for?